apps/mobile-wallet/src/navigation/RootStackNavigation.tsx

```diff
@@ -275,14 +266,8 @@ const AppUnlockModal = ({ initialRouteName }: Required<RootStackNavigationProps>
   useAutoLock(unlockApp)
 
   return (
-    <Modal
-      visible={!!lastUsedWalletId && biometricsRequiredForAppAccess && !isWalletUnlocked}
-      onLayout={handleScreenLayoutChange}
-      animationType="none"
-    >
-      <View style={{ backgroundColor: 'black', flex: 1 }}>
-        <CoolAlephiumCanvas {...dimensions} onPress={unlockApp} />
```

Comment:
The problem with removing the canvas is that we lose the feature of re-triggering the unlock flow `onPress`.

Reply:
If a user cancels the authentication for whatever reason, the authentication modal goes away and they have no way of triggering it. That's why we had the `onPress` handler that apart from the cool animation of flipping the Alephium logo, it also re-triggered the authentication flow.

Reply:
Good point, I can add the onPress handler on the black view (and add an alephium logo). But the allowed me to find a bug.. In the following modal, if I tap on try again nothing happens. If I tap on cancel, it shows an error but let me try again :D 

![image](https://github.com/user-attachments/assets/5d96b0fc-8fa6-467c-b0d6-4dce3f578e68)

Cancel:
![image](https://github.com/user-attachments/assets/8598b72c-40ef-4946-9c75-7660c3e6ccdc)

Reply:
Tap to unlock re-add here 844069d9983fdbdcdd4538c43800de991acc8c60

Reply:
I sent you a msg on Slack on how to potentially solve this build error